assets/js/modules/analytics-4/components/widgets/ConnectGA4CTATileWidget.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * ConnectGA4CTATileWidget component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import AnalyticsIcon from '../../../../../svg/graphics/analytics.svg';
+import ConnectModuleCTATile from '../../../../components/KeyMetrics/ConnectModuleCTATile';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import {
+	CORE_USER,
+	keyMetricsGA4Widgets,
+} from '../../../../googlesitekit/datastore/user/constants';
+
+const { useSelect } = Data;
+
+export default function ConnectGA4CTATileWidget( { Widget, WidgetNull } ) {
+	const isGA4ModuleConnected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'analytics-4' )
+	);
+
+	const keyMetrics = useSelect( ( select ) => {
+		if ( isGA4ModuleConnected !== false ) {
+			return;
+		}
+		return select( CORE_USER ).getKeyMetrics();
+	} );
+
+	let hideWidget = true;
+
+	if ( keyMetrics?.length > 0 ) {
+		const kmAnalyticsWidgetCount = keyMetrics.filter( ( keyMetric ) =>
+			keyMetricsGA4Widgets.includes( keyMetric )
+		).length;
+
+		if ( kmAnalyticsWidgetCount > 0 && kmAnalyticsWidgetCount < 3 ) {
+			hideWidget = false;
+		}
+	}
+
+	if ( isGA4ModuleConnected !== false || hideWidget ) {
+		return <WidgetNull />;
+	}
+
+	return (
+		<ConnectModuleCTATile
+			Icon={ AnalyticsIcon }
+			moduleName={ __( 'Google Analytics', 'google-site-kit' ) }
```

Comment:
Let's probably get the module name from the module information that we store in the `core/modules` store. Just use `getModule( moduleSlug )` selector in the `ConnectModuleCTATile` component to get the module info and use `module.name` property. Once we do this, we will no longer need this `mouldeName` property and can remove it.

Reply:
I wanted to do this initially but this will not work, as Analytics is referred as Google Analytics in Figma but the `module.name` is Just Analytics. So, I think we'll need the module name prop or have to divert from the design. 

````js
googlesitekit.data.select('core/modules').getModule('adsense').name //'AdSense'
googlesitekit.data.select('core/modules').getModule('analytics').name // 'Analytics'
googlesitekit.data.select('core/modules').getModule('analytics-4').name //'Analytics 4'
````

Thoughts?

Reply:
I think that's fine to have just `Analytics` there since that is what we use everywhere in the plugin. Let's stick with module names for now.